src/components/EpisodesNav/renderUtils.jsx

```diff
@@ -37,10 +37,28 @@ function renderPaginationLink(data, index){
 
 	const episodeId = UTILS.convertToSafeString(title, "-");
 
-	const isActive = index === activeIndex;
+	const isActive 	= index === activeIndex;
+
+	let opacity = 1;
+	if(!isActive){
+		const decayDistance = 4;
+		const decayRate     = 1/decayDistance;
+		const fadeOffset    = Math.abs(index - activeIndex);
+		
+		opacity -= fadeOffset * decayRate;
+	}
+	
+	const offset = index - activeIndex;
+
+	const style = {
+		transform: `translateX(${offset}em)`,
+		opacity: opacity
```

Comment:
```suggestion
		opacity
```
If the key / value names are the same then you don't need to specify both sides

Reply:
Ah, forgot that